Code/Tools/ProjectManager/Source/GemRepo/GemRepoInspector.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <GemRepo/GemRepoInspector.h>
+#include <GemRepo/GemRepoItemDelegate.h>
+
+#include <QFrame>
+#include <QLabel>
+#include <QVBoxLayout>
+#include <QIcon>
+
+namespace O3DE::ProjectManager
+{
+    GemRepoInspector::GemRepoInspector(GemRepoModel* model, QWidget* parent)
+        : QScrollArea(parent)
+        , m_model(model)
+    {
+        setObjectName("gemRepoInspector");
+        setWidgetResizable(true);
+        setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
+        setVerticalScrollBarPolicy(Qt::ScrollBarAsNeeded);
+
+        m_mainWidget = new QWidget();
+        setWidget(m_mainWidget);
+
+        m_mainLayout = new QVBoxLayout();
+        m_mainLayout->setMargin(15);
+        m_mainLayout->setAlignment(Qt::AlignTop);
+        m_mainWidget->setLayout(m_mainLayout);
+
+        InitMainWidget();
+
+        connect(m_model->GetSelectionModel(), &QItemSelectionModel::selectionChanged, this, &GemRepoInspector::OnSelectionChanged);
+        Update({});
+    }
+
+    void GemRepoInspector::OnSelectionChanged(const QItemSelection& selected, [[maybe_unused]] const QItemSelection& deselected)
+    {
+        const QModelIndexList selectedIndices = selected.indexes();
+        if (selectedIndices.empty())
+        {
+            Update({});
+            return;
+        }
+
+        Update(selectedIndices[0]);
+    }
+
+    void GemRepoInspector::Update(const QModelIndex& modelIndex)
+    {
+        if (!modelIndex.isValid())
+        {
+            m_mainWidget->hide();
+        }
+
+        // Repo name and url link
+        m_nameLabel->setText(m_model->GetName(modelIndex));
+        m_repoLinkLabel->setText(m_model->GetRepoLink(modelIndex));
+        m_repoLinkLabel->SetUrl(m_model->GetRepoLink(modelIndex));
+
+        // Repo summary
+        m_summaryLabel->setText(m_model->GetSummary(modelIndex));
+        m_summaryLabel->adjustSize();
+
+        // Additional information
+        if (m_model->HasAdditionalInfo(modelIndex))
+        {
+            m_addInfoTitleLabel->show();
+            m_addInfoTextLabel->show();
+
+            m_addInfoSpacer->changeSize(0, 20, QSizePolicy::Fixed, QSizePolicy::Fixed);
+
+            m_addInfoTextLabel->setText(m_model->GetAdditionalInfo(modelIndex));
+        }
+        else
+        {
+            m_addInfoTitleLabel->hide();
+            m_addInfoTextLabel->hide();
+
+            m_addInfoSpacer->changeSize(0, 0, QSizePolicy::Fixed, QSizePolicy::Fixed);
+        }
+
+        // Included Gems
+        m_includedGems->Update("Included Gems", "", m_model->GetIncludedGemNames(modelIndex));
```

Comment:
Use `tr()` for display strings

Reply:
Shoot I spaced out on tr here thanks for catching it!